code/modules/reagents/chemistry/holder.dm

```diff
@@ -211,36 +211,44 @@ var/const/INGEST = 2
 			continue
 		if(!M)
 			M = R.holder.my_atom
+
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
 			//Check if this mob's species is set and can process this type of reagent
 			var/can_process = 0
```

Comment:
Yes, because this doesn't change what reagents can be processed by a species, but rather what effects they receive from those they can.

I almost removed this initially, but realized that required a lot of "empty" overrides for reagents IPCs don't process where we just remove the reagent